src/pages/personnel/duck/Reducers.js

```diff
@@ -0,0 +1,38 @@
+import { combineReducers } from "redux";
+
+import types from "./Types";
+
+const defaultState = {
+    personnel: [],
+    clients: [],
```

Comment:
Use null for personnel and clients default state and display loading page in page